recipe/meta.yaml

```diff
@@ -1,59 +1,54 @@
 {% set name = "prophet" %}
-{% set version = "1.1.4" %}
+{% set version = "1.1.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7f91e4ba322786e9512158783800b6eafb002200a7b3e9eadc13d7ff4d6e0634
-  patches:
-    - setup-make-folder.patch
+  sha256: 80973c0b8a22d835bfa9d6665a78ebc63115135eaef0f73b46ee14e9bad3ca1a
 
 build:
   number: 0
   # LunarCalendar, which is a noarch package, requires ephem, unavailable on s390x
   # https://blog.quantinsti.com/installing-prophet-library-windows/
   skip: true  # [py<37 or (linux and s390x)]
+  # because of missing numpy
+  skip: true  # [py>311]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   binary_has_prefix_files:
     - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.bin
   # To use our own cmdstan rather than download and build a different one:
   script_env:
     - PROPHET_REPACKAGE_CMDSTAN=false
+    - PRECOMPILED_HEADERS=false  # [win]
     - BOOST="${LIBRARY_INC}"  # [win]
+  missing_dso_whitelist:
+    - lib/libtbb.12.dylib  # [osx]
+    - lib/libtbb.so.12      # [linux]
+    - libgfortran.so.5.0.0  # [linux]
```

Comment:
These three lib seem wrong to me. Why are they missing?

Reply:
I mistakenly added `libgfortran`. I removed it. `libttb` is necessary though. Thanks for catching that.